egs2/iwslt22_low_resource/st1/run.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+src_lang=ta
+tgt_lang=fr
+
+# train: 17 hours of Tamasheq audio data aligned to French translations
+# train_full: a 19 hour version of this corpus, 
+# including 2 additional hours of data that was labeled by annotators as potentially noisy 
+train_set=train
+train_dev=valid
+test_set=test
+
+st_config=conf/train_st_transformer.yaml
+inference_config=conf/decode_st_transformer.yaml
+
+tgt_nbpe=1000
+
+# tc: truecase
+# lc: lowercase
+# lc.rm: lowercase with punctuation removal
+# Note, it is best to keep tgt_case as tc to match IWSLT22 eval
+tgt_case=tc
+
+./st.sh \
+		--st_tag wav2vec-transformer-warmup-15k \
+    --ignore_init_mismatch true \
```

Comment:
```suggestion
```

Reply:
This is not necessary since you do not use any pre-trained model with mismatched keys